client/lib/route/page-notifier.js

```diff
@@ -1,23 +1,34 @@
 /*eslint no-multi-spaces: 0*/
 // node dependency
 var EventEmitter   = require( 'events' ).EventEmitter;
+var ipc = require( 'electron' ).ipcRenderer;
+var includes = require( 'lodash' ).includes;
 
 // internal dependency
-var page           = require( 'page' );
-var Route          = page.Route;
+var page		   = require( 'page' );
+var Route		  = page.Route;
 
 // routing state and middleware
-var location       = new EventEmitter();
+var location	   = new EventEmitter();
 var currentContext = null;
-var route          = new Route( '*' );
+var commandLine	= {};
+var route		  = new Route( '*' );
 
 // Inject our route at the front so we are always called
 page.callbacks.unshift( route.middleware( function( context, next ) {
+	context.commandLine = commandLine;
 	next();
 	location.emit( 'change', context, currentContext );
 	currentContext = context;
 } ) );
 
+ipc.on( 'command-line-arguments', function( sender, args ) {
+	commandLine = args;
+	if ( includes( args._, 'post' ) ) {
```

Comment:
Could this be `createPost` instead, considering 'post' is somewhat ambiguous (verb, noun, method)?